<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Rdiffweb-discuss] handle new dirs,	500 internal error and	UserRoot
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/rdiffweb-discuss/2007-April/index.html" >
   <LINK REL="made" HREF="mailto:rdiffweb-discuss%40lists.berlios.de?Subject=Re%3A%20%5BRdiffweb-discuss%5D%20handle%20new%20dirs%2C%0A%09500%20internal%20error%20and%09UserRoot&In-Reply-To=%3C1175751386.5799.4.camel%40carnivour.artus.own%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000118.html">
   <LINK REL="Next"  HREF="000126.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Rdiffweb-discuss] handle new dirs,	500 internal error and	UserRoot</H1>
    <B>Christoph Langbein</B> 
    <A HREF="mailto:rdiffweb-discuss%40lists.berlios.de?Subject=Re%3A%20%5BRdiffweb-discuss%5D%20handle%20new%20dirs%2C%0A%09500%20internal%20error%20and%09UserRoot&In-Reply-To=%3C1175751386.5799.4.camel%40carnivour.artus.own%3E"
       TITLE="[Rdiffweb-discuss] handle new dirs,	500 internal error and	UserRoot">christoph at greatescape.de
       </A><BR>
    <I>Thu Apr  5 07:36:26 CEST 2007</I>
    <P><UL>
        <LI>Previous message: <A HREF="000118.html">[Rdiffweb-discuss] handle new dirs,	500 internal error and UserRoot
</A></li>
        <LI>Next message: <A HREF="000126.html">[Rdiffweb-discuss] handle new dirs,	500 internal error and	UserRoot
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#124">[ date ]</a>
              <a href="thread.html#124">[ thread ]</a>
              <a href="subject.html#124">[ subject ]</a>
              <a href="author.html#124">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Am Mittwoch, den 04.04.2007, 17:23 -0400 schrieb Josh Nisly:
&gt;<i> Christoph Langbein wrote:
</I>&gt;<i> &gt; Hello,
</I>&gt;<i> &gt; to go like a bull at a gate: I'm having trouble with the current svn
</I>&gt;<i> &gt; version. 
</I>&gt;<i> &gt; If there is a path for a directory configured in rdw.conf that has never
</I>&gt;<i> &gt; been initialized (never run rdiff-backup).
</I>&gt;<i> &gt; Maybe this is the 500 Internal error message asked here in the list in
</I>&gt;<i> &gt; march.
</I>&gt;<i> &gt; I get this error messages:
</I>&gt;<i> &gt; Page handler: &lt;bound method rdiffLocationsPage.index of
</I>&gt;<i> &gt; &lt;rdiffWeb.page_locations.rdiffLocationsPage instance at 0xa78af34c&gt;&gt;
</I>&gt;<i> &gt; Traceback (most recent call last):
</I>&gt;<i> &gt;   File &quot;/var/lib/python-support/python2.4/cherrypy/_cphttptools.py&quot;, line 105, in _run
</I>&gt;<i> &gt;     self.main()
</I>&gt;<i> &gt;   File &quot;/var/lib/python-support/python2.4/cherrypy/_cphttptools.py&quot;, line 254, in main
</I>&gt;<i> &gt;     body = page_handler(*virtual_path, **self.params)
</I>&gt;<i> &gt;   File &quot;/usr/lib/python2.4/site-packages/rdiffWeb/page_locations.py&quot;, line 13, in index
</I>&gt;<i> &gt;     repoHistory = librdiff.getLastBackupHistoryEntry(rdw_helpers.joinPaths(self.userDB.getUserRoot(self.getUsername()), userRepo))
</I>&gt;<i> &gt;   File &quot;/usr/lib/python2.4/site-packages/rdiffWeb/librdiff.py&quot;, line 287, in getLastBackupHistoryEntry
</I>&gt;<i> &gt;     return _getBackupHistory(repoRoot, 1)[0]
</I>&gt;<i> &gt; IndexError: list index out of range
</I>&gt;<i> &gt;
</I>&gt;<i> &gt; would be nice if this could be catched, as the other directories could be displayed.
</I>&gt;<i> &gt;   
</I>&gt;<i> I can't duplicate this. There is extensive validation in place to make 
</I>&gt;<i> sure that this doesn't happen. Can you give more information on the 
</I>&gt;<i> directory that is triggering the error? Does the directory exist? If so, 
</I>&gt;<i> are there any subdirectories?
</I>directory exists. The tree looks like
backup1/dir1
backup1/dir1/file_1
backup1/dir1/file_n
backup1/dir1/dir_n
backup1/dir2/file_1
backup1/dir1/file_n
backup1/dir1/dir_n
backup1/rdiff-backup-data

and rdiff-backup IS running the first time !

Christoph


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000118.html">[Rdiffweb-discuss] handle new dirs,	500 internal error and UserRoot
</A></li>
	<LI>Next message: <A HREF="000126.html">[Rdiffweb-discuss] handle new dirs,	500 internal error and	UserRoot
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#124">[ date ]</a>
              <a href="thread.html#124">[ thread ]</a>
              <a href="subject.html#124">[ subject ]</a>
              <a href="author.html#124">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/rdiffweb-discuss">More information about the Rdiffweb-discuss
mailing list</a><br>
</body></html>
